docs/staticwebapp.config.json

```diff
@@ -1,13 +1,13 @@
 {
 	"routes": [
 		{
-			"route": "/docs/fcdocstest",
+			"route": "/docs/fcdocstest/",
 			"redirect": "/docs/apis",
 			"statusCode": 301
 		},
 		{
-			"route": "/docs/apis/*",
-			"redirect": "/docs/apis",
+			"route": "/docs/apis/{*}",
+			"redirect": "/docs/apis/{*}",
```

Comment:
I believe we should not have any changes to this line as this is the redirect location